tests/test-artifacts/in-memory-connectors-test-artifact/src/main/java/org/wso2/carbon/identity/mgt/store/connector/inmemory/InMemoryIdentityStoreConnector.java

```diff
@@ -1,5 +1,5 @@
 /*
-* Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+* Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
No need to update the license year if you are not introducing a new file.

Reply:
Fixed with https://github.com/wso2/carbon-identity-mgt/pull/182